src/osgPlugins/OpenFlight/MaterialPaletteManager.h

```diff
@@ -32,7 +32,7 @@ namespace flt
 class DataOutputStream;
 
 
-class MaterialPaletteManager : public osg::Referenced
```

Comment:
Let's take the 3.6 version of this file.

Reply:
Reverted change to this file (pending)